app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
```

Comment:
- [ ] [OPTIONAL] Based on the[ UML ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) diagram provided, please kindly consider that you do not need to use a foreign key 'author_id' on comments.